openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# openedx.yaml
+
+---
+owner:
+    type: team
+    team: edx/teaching-and-learning
```

Comment:
@nedbat: Is there a canonical team names list? TNL is the right group, but I don't know if we're supposed to be writing this vs. `edx/platform/tnl-squad`, etc.

Reply:
It's supposed to be a GitHub team, so it's already correct: https://github.com/orgs/edx/teams/teaching-and-learning/members